Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -457,6 +457,31 @@ static ErrorOr<void, DlErrorMessage> link_main_library(int flags, DependencyOrde
     return {};
 }
 
```

Comment:
> While dlclose is not used often in the wild (and we certainly don't have references to it in Serenity) and is not required by POSIX to do anything, I would rather copy glibc's behavior instead of leaving musl-like no-op behavior.

Why tho?
I'm sorry, but I read that as:
> While dlclose doing anything is literally useless, I want to make it do something.